src/lang/en.json

```diff
@@ -651,6 +655,11 @@
   "teamfight_participation": "Radiant participation/Dire participation",
   "teamfight_score": "Radiant kills/Dire kills",
   "teamfight_radiant_gold_adv": "Net gold advantage for Radiant",
-  "teamfight_radiant_xp_adv": "Net XP advantage for Radiant"
+  "teamfight_radiant_xp_adv": "Net XP advantage for Radiant",
 
+  "vision_expired": "Expired after",
+  "vision_destroyed": "Destroyed after",
+  "vision_all_time": "All time",
+  "vision_placed_observer": "placed Observer at",
```

Comment:
you might want to double check with @kozhilya or another translator but you might want to use a template string here (see time strings) to avoid word order issues in other languages.